src/App.css

```diff
@@ -37,30 +37,84 @@ ul{
     list-style: none;
 }
 .dark-theme{
-    background-color: var(--background-slate);
+    background-color: var(--background-dark);
     color: var(--font-secondary);
 }
 
 .day-theme{
     background-color: var(--background-light);
     color: var(--font-dark);
 }
+
+.dark-fill{
+    background-color: var(--font-dark);
+    color: var(--font-secondary);
+    border: none;
+}
+
+.light-fill{
+    background-color: var(--font-secondary);
+    color: var(--font-dark);
+    border: none;
+}
+
+.mono-btn{
+    font-size: 1.2rem;
+    padding: 1rem;
+}
+
+.outline-btn-day{
+    background-color: transparent;
+    color: var(--background-dark);
+    border: 1px solid var(--background-dark);
+}
+
+
+.outline-btn-dark{
+    background-color: transparent;
+    border: 1px solid var(--background-light);
+    color: var(--background-light);
+}
+
 .hide{
     display: none;
 }
 .show{
     display: block;
 }
 
+.pointer{
+    cursor: pointer;
+}
+
+.btn-primary-outline{
+    color: inherit;
+    border-color: inherit;
+}
+
+.btn-primary-outline:hover{
+    color: inherit;
+    background-color: transparent;
+    border-color: inherit;
```

Comment:
Liked how you've inherited the colors here, would make adding a dark theme easier.
Good Work!